crypto/bn/bn_mont.c

```diff
@@ -367,6 +385,11 @@ int BN_MONT_CTX_set(BN_MONT_CTX *mont, const BIGNUM *mod, BN_CTX *ctx)
     if (!BN_mod(&(mont->RR), &(mont->RR), &(mont->N), ctx))
         goto err;
 
+    for (i = mont->RR.top, ret = mont->N.top; i < ret; i++)
+        mont->RR.d[i] = 0;
+    mont->RR.top = ret;
+    mont->RR.flags |= BN_FLG_FIXED_TOP;
```

Comment:
Might be worth a comment in bn_lcl.h where `mont->RR` is defined that it is non-minimal and thus can't be passed to most `BIGNUM` functions.

(Hrm, the `RSAZ_ENABLED` code seems to have already assumed this one was padded up...)